### PR TITLE
Added calculation of discontinuity sequence when there's an existing discontinuity in the vod manifest

### PR DESCRIPTION
**Pull request by mrocajr**

### CODE REVIEW

File: live-hls.js [hunk 1]
```diff
@@ -34,7 +34,11 @@ var defaults = {
   manifestnotfound: 0,
```
Comment (ldayananda) [inline live-hls.js:34]: looks like a mis-indent from here to line 44

File: live-hls.js [hunk 2]
```diff
@@ -325,7 +341,15 @@ const extractResourceWindow = function(mfest, duration, event, streamtype) {
   debuglog('start before mod ' + startposition);
   debuglog('event start ' + event.start);
 
-  event.discontinuity = Math.floor(startposition / resource.length);
+  event.discomod = 0;
+  for (i = 0; i < event.discotrack.length; i++) {
+    let rlength = resource.length - 1;
+    event.discomod += Math.floor((startposition + resource.length - 1 - event.discotrack[i]) / (resource.length) );
+    console.log('start: ' + startposition + ' + rlength: ' + resource.length + '-1 / rlength: ' + resource.length + 'disco: ' + event.discotrack[i] +' = ' +event.discomod);
```
Comment (ldayananda) [inline live-hls.js]: did you mean to use `rlength` in the calc and get rid of the console.log?

File: live-hls.js [hunk 3]
```diff
@@ -382,7 +413,11 @@ const extractResourceWindow = function(mfest, duration, event, streamtype) {
         var referencedResource = i % resource.length;
 
         if (referencedResource === 0) {
-          lines.push('#EXT-X-DISCONTINUITY');
+          resource[i].disco = '#EXT-X-DISCONTINUITY';
+        }
+
+        if (resource[i].disco) {
+          lines.push(resource[i].disco);
```
Comment (ldayananda) [inline live-hls.js:417]: why not just do this inside the previous if on line 417?

File: live-hls.js [hunk 4]
```diff
@@ -325,7 +341,15 @@ const extractResourceWindow = function(mfest, duration, event, streamtype) {
   debuglog('start before mod ' + startposition);
   debuglog('event start ' + event.start);
 
-  event.discontinuity = Math.floor(startposition / resource.length);
+  event.discomod = 0;
+  for (i = 0; i < event.discotrack.length; i++) {
+    let rlength = resource.length - 1;
+    event.discomod += Math.floor((startposition + resource.length - 1 - event.discotrack[i]) / (resource.length) );
```
Comment (ldayananda) [inline live-hls.js:347]: a comment on what this math is doing would be helpful

**Review by ldayananda — COMMENTED** [review]
testing now

File: live-hls.js (hunk 2 above)
Comment (mrocajr) [reply live-hls.js]: Oops, I forgot to get rid of the console log. It's probably easier if I get rid of rlength as well.

File: live-hls.js (hunk 3 above)
Comment (mrocajr) [reply live-hls.js:417]: Good call.

**Review by ldayananda — APPROVED** [review]

**Pull request merged**